lib/parslet/context.rb

```diff
@@ -32,4 +24,31 @@ def initialize(bindings)
       instance_variable_set("@#{key}", value)
     end
   end
+
+  # following methods are needed and were provided by blankslate
```

Comment:
hmm, you manage to make that (my) effort sound very second rate. And also like you don't really want it at all. I do hope i am getting the wrong impression.

To "just" make tests pass was intentional.  I mean to make the tests pass per se, i consider a good thing. In fact given that all tests pass, i consider it working. All tests passing being the definition of working software. (Off course that may mean there are too little tests, but i don't think that is the case here)

So, on to the "just" : I had noticed the test that demands the method implementation. And opted for leaving the test in and fixing the pr to pass it. Leaving it at your discretion to remove both test and  method (which is what i would have done, but i have only partial understanding of the system). I was doing this as you are not exactly easy going about accepting pr's and i thought not changing the tests would be better in your view. Apparently my misconception.

About the methods working: as per docs http://ruby-doc.org/core-1.9.3/BasicObject.html they are all implemented. I didn't "just" write the method names there if that's what you thought. I don't see how it would pass the tests otherwise.

BasicObject has no respond_to, which is obviously not used in the tests or the code, so maybe not needed ?